static/css/main.css

```diff
@@ -940,13 +967,14 @@ blockquote p cite a:hover {
 }
 
 .navbar-light .navbar-nav li:hover {
-    border-bottom: var(--mesh-dk-blue-ultralight) 1px solid;
+    border-bottom: var(--mesh-blue) 2px solid;
```

Comment:
The px dimensions need to be consistent. When you hover your mouse, it jumps the font bar up and down. There's a clear one that balances this. right now it jitters